src/ucp/api/ucp.h

```diff
@@ -1198,8 +1197,6 @@ ucs_status_t ucp_get_nbi(ucp_ep_h ep, void *buffer, size_t length,
  * @a rkey. The @a add value is the value that is used for the add operation.
  * When the operation completes the sum of the original remote value and the
  * operand value (@a add) is stored in remote memory.
- * The call to the routine returns immediately, independent of operation
- * completion.
```

Comment:
In the current implementation the routine will not return until required resources are available (for instance cq entries or tx memory chunks). But we can't say that the operation is completed remotely, cause it's not really the case for some of the transports.